### PR TITLE
zh: add pd-recover doc

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)

add pd-recover doc
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [x] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/pd-recover.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+示例：
+
+```
+kubectl get tc test -n test -o='go-template={{.status.clusterID}}{{"\n"}}'
+6821434242797747735
+```
+
+### 获取 Alloc ID
+
+Alloc ID 是 TiKV 或者 TiFlash 分配的 StoreID。
```
Comment (tennix) [inline zh/pd-recover.md:52]: alloc-id is generated by a global id allocator, it's used in store id, peer id, region id etc. So we cannot use the store id to decide the current alloc id.

**Comment by DanielZhangQD** [comment]
@tennix comments addressed, PTAL again.

File: zh/pd-recover.md (hunk 1 above)
Comment (DanielZhangQD) [reply zh/pd-recover.md]: updated

File: zh/pd-recover.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
+---
+title: 使用 PD Recover 恢复 PD 集群
+summary: 使用 PD Recover 恢复 PD 集群
```
Comment (lilin90) [inline zh/pd-recover.md:3]:
```suggestion
summary: 了解如何使用 PD Recover 恢复 PD 集群。
```

File: zh/pd-recover.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+category: reference
+---
+
+# 使用 PD Recover 恢复 PD 集群
+
+[PD Recover](https://pingcap.com/docs-cn/stable/reference/tools/pd-recover) 是对 PD 进行灾难性恢复的工具，用于恢复无法正常启动或服务的 PD 集群。
+
+## 下载 pd-recover
```
Comment (lilin90) [inline zh/pd-recover.md:11]:
```suggestion
## 下载 PD Recover
```

File: zh/pd-recover.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+下载 TiDB 官方安装包：
+
+{{< copyable "shell-regular" >}}
+
+```
```
Comment (lilin90) [inline zh/pd-recover.md:17]:
```suggestion
```shell
```

File: zh/pd-recover.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+wget https://download.pingcap.org/tidb-${version}-linux-amd64.tar.gz
+```
+
+`${version}` 是 TiDB 集群版本，例如，`v4.0.0-rc`。
+
+解压安装包：
+
+{{< copyable "shell-regular" >}}
+
+```
```
Comment (lilin90) [inline zh/pd-recover.md:27]:
```suggestion
```shell
```

File: zh/pd-recover.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+tar -xzf tidb-${version}-linux-amd64.tar.gz
+```
+
+`pd-recover` 在 `tidb-${version}-linux-amd64/bin` 目录下。
+
+## 使用 pd-recover 恢复 PD 集群
```
Comment (lilin90) [inline zh/pd-recover.md:33]:
```suggestion
## 使用 PD Recover 恢复 PD 集群
```

**Comment by DanielZhangQD** [comment]
@lilin90 comments addressed, PTAL again.

**Pull request merged**